src/components/Kv/KvButton.vue

```diff
@@ -75,6 +91,10 @@ export default {
 		}
 	}
 
+	&.warning {
```

Comment:
New style for red button. I wasn't sure if this should be `.warning` (like .secondary) or `--red` modifier

Reply:
I think maybe this should be `alert` instead of `warning` based on the color paradigms set forth from Foundation. We have other warning styles in place and they are all a specified shade of yellow.

https://get.foundation/sites/docs/global.html#colors